pytest_splunk_addon/standard_lib/requirement_tests/test_generator.py

```diff
@@ -103,8 +103,10 @@ def strip_syslog_header(self, raw_event):
         if regex_rfc5424:
             stripped_header = regex_rfc5424.group(3)
             return stripped_header
+        #    regex = r"([A-Z][a-z][a-z]\s{1,2}\d{1,2}\s\d{2}[:]\d{2}[:]\d{2})\s+([\w][\w\d\.@-]*)\s\w*:?(.*)$",
+        #   (?:\s\d{4})? Added to support ipv6 cisco asa date
```

Comment:
The root of the problem is the timestamp formatting, not matter if ASA uses ipv4/ipv6 - 
![image](https://user-images.githubusercontent.com/83703180/163353589-75b89b59-f5bb-4776-89d4-5a844405bfc6.png)